examples/jsm/renderers/webgpu/WebGPUTextures.js

```diff
@@ -732,6 +716,50 @@ class WebGPUTextures {
 
 	}
 
+	_isHTMLImage( image ) {
+
+		return ( typeof HTMLImageElement !== 'undefined' && image instanceof HTMLImageElement ) || ( typeof HTMLCanvasElement !== 'undefined' && image instanceof HTMLCanvasElement );
+
+	}
+
+	_copyImageToTexture( image, texture, textureGPU, textureGPUDescriptor, needsMipmaps, originDepth ) {
+
+		if ( this._isHTMLImage( image ) ) {
+
+			this._getImageBitmapFromHTML( image, texture ).then( imageBitmap => {
+
+				this._copyExternalImageToTexture( imageBitmap, textureGPU, originDepth );
+
+				if ( needsMipmaps ) this._generateMipmaps( textureGPU, textureGPUDescriptor, originDepth );
```

Comment:
Why not also move `generateMipmaps` to `copyExternalImageToTexture`, like with the previous funtions?

Reply:
Yes, good catch as usual :)